### PR TITLE
Trigger function for vitals fix (CU-86dtk5h8k)

### PR DESCRIPTION
**Pull request by Darrickg**
Added migration script 44 to fix the function

Test Plan

- [x] Passes npm run test
- [x] Passes linter
- [x]  Works in dev deployment

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86dtk5h8k Investigate why some buttons don't send RSSI and SNR vitals](https://app.clickup.com/t/86dtk5h8k)

File: server/db/044-fixtriggerfunction.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            ON CONFLICT (device_id)
+            DO UPDATE SET
+                id = NEW.id,
+                battery_level = NEW.battery_level,
+                created_at = NEW.created_at,
+                rssi = NEW.rssi,
+                snr = NEW.snr;
+            RETURN NEW;
+        END;
+	    $t$;
```
Comment (micahdbak) [inline server/db/044-fixtriggerfunction.sql:33]: minus 4 spaces here

**Review by micahdbak — COMMENTED** [review]
In the 30 minutes between devices and the deployment, please deploy this to the chatbot dev server and confirm there are no error messages! ++ On your local database.

File: server/db/044-fixtriggerfunction.sql (hunk 1 above)
Comment (Darrickg) [reply server/db/044-fixtriggerfunction.sql:33]: fixed in newest change

**Comment by Darrickg** [comment]
should probably close the investigation branch as it's filled with a bunch of extra logs and notices for debugging but no real changes

**Review by micahdbak — APPROVED** [review]
Looks good!

FYI: thanks for deploying to dev with me on call :)

**Pull request merged**